specs/interop/messaging.md

```diff
@@ -136,13 +149,121 @@ This feature could be added in a backwards compatible way by adding a new functi
 One possible way to handle explicit denial of service attacks is to utilize identity
 in iterated games such that the block builder can ban identities that submit malicious transactions.
 
-### Message Expiry
+### Message Expiry Invariant
+
+Note: Message Expiry as property of the protocol is in active discussion.
```

Comment:
We cannot ignore this invariant but we can relax it and we do so via making `EXPIRY_TIME = 0`. In practice this applies in the case of a reorg, to ensure that the executing message always happens at the same time or in the future

Reply:
Ah i see this is in addition to the timestamp invariant